api/db/seeds/data/common/constants.js

```diff
@@ -0,0 +1,9 @@
+const FEATURE_OFFEST_ID = 1000;
+
+const FEATURE_COMPUTE_ORGANIZATION_LEARNER_CERTIFICABILITY_OFFSET_ID = FEATURE_OFFEST_ID + 1;
+const FEATURE_MULTIPLE_SENDING_ASSESSMENT_OFFSET_ID = FEATURE_OFFEST_ID + 2;
```

Comment:
```suggestion
const FEATURE_COMPUTE_ORGANIZATION_LEARNER_CERTIFICABILITY_ID = FEATURE_OFFEST_ID + 1;
const FEATURE_MULTIPLE_SENDING_ASSESSMENT_ID = FEATURE_OFFEST_ID + 2;
```
Je sais pas si ici c'est pas juste l'id qu'on veut et pas le offset dans le nommage ?